delete-user-data/extension.yaml

```diff
@@ -106,8 +106,9 @@ params:
     type: select
     label: Cloud Firestore delete mode
     description: >-
-      How do you want to delete Cloud Firestore documents? To also delete documents in
-      subcollections, set this parameter to `recursive`.
+      Only applicable if you use the `Cloud Firestore paths` parameter. How do you want
```

Comment:
Should the first sentence be inside parentheses?

Reply:
sure!